ai_diffusion/style.py

```diff
@@ -210,7 +203,7 @@ def get_sampler_config(self, is_upscaling=False, is_live=False):
         if is_live:
             return SamplerConfig(self.live_sampler, self.live_sampler_steps, self.live_cfg_scale)
         if is_upscaling:
```

Comment:
this branch (and the parameter) are no longer useful now

Reply:
Yep, removed.